code/modules/overmap/objects/outpost/outpost.dm

```diff
@@ -41,6 +41,12 @@
 	var/max_missions = 15
 	/// List of missions that can be accepted at this outpost. Missions which have been accepted are removed from this list.
 	var/list/datum/mission/missions
+	/// List of all of the things this outpost offers
+	var/list/supply_packs = list()
+	/// our 'Order number'
+	var/ordernum = 1
+	/// Our faction of the outpost
+	var/faction
```

Comment:
Both seem to be unused, supply_pack still looks like it uses SSshuttle's ordernum

Reply:
supply pack SHOULD use ordernum  now